lib/livebook_web/live/session_live/mix_standalone_live.ex

```diff
@@ -31,6 +31,12 @@ defmodule LivebookWeb.SessionLive.MixStandaloneLive do
         This way all your code and dependencies will be available
         within the notebook.
       </p>
+      <p class="text-gray-700">
+        <span class="font-semibold">Warning:</span>
+        Notebooks that use <code>Mix.install</code> does not work
```

Comment:
Note: the `code` tag is not gonna use the usual styles as these apply only within `.markdown` container. The easiest option is to add `markdown` to the paragraph class.

Reply:
It looks good enough to me (the font is monospaced), so I am ok with shipping as is, unless you have reservations?

Reply:
```suggestion
        Notebooks that use <code>Mix.install/1</code> do not work
```

Reply:
This reads much better to me:

![image](https://user-images.githubusercontent.com/17034772/121377087-5f5eb980-c942-11eb-8509-5c17ce8d5916.png)

Subjective though :)

Reply:
Yeah, I would be comfortable with adding Markdown if we were actually writing those in markdown but, given we are not, I will keep things simple. :)